instrumentation/runtime-telemetry/runtime-telemetry-java17/library/src/main/java/io/opentelemetry/instrumentation/runtimemetrics/java17/internal/threads/ThreadCountHandler.java

```diff
@@ -21,7 +21,7 @@
  * any time.
  */
 public final class ThreadCountHandler implements RecordedEventHandler {
-  private static final String METRIC_NAME = "process.runtime.jvm.threads.count";
+  private static final String METRIC_NAME = "jvm.thread.count";
```

Comment:
Plural:

```suggestion
  private static final String METRIC_NAME = "jvm.threads.count";
```

Reply:
As in case of "jvm.class.loaded" the specification uses singular form "thread"